# HG changeset patch
# User Steve Borho <steve@borho.org>
# Date 1362690570 21600
# Branch stable
# Node ID e571f9b3eadc2e2706bd350dab33d3bee5bf2315
# Parent  6fbbe3d1974c267e5a30e4d9d1efc54a24fd1f35
qt: use int comparison for Qt and PyQt version check (fixes #2467)

diff --git a/tortoisehg/hgqt/qtlib.py b/tortoisehg/hgqt/qtlib.py
--- a/tortoisehg/hgqt/qtlib.py
+++ b/tortoisehg/hgqt/qtlib.py
@@ -25,9 +25,8 @@
 from PyQt4.QtCore import *
 from PyQt4.QtGui import *
 
-if PYQT_VERSION_STR.split('.') < ['4', '6'] or \
-   QT_VERSION_STR.split('.') < ['4', '6']:
-    sys.stderr.write('TortoiseHg requires Qt 4.6 and PyQt 4.7\n')
+if PYQT_VERSION < 0x40600 or QT_VERSION < 0x40600:
+    sys.stderr.write('TortoiseHg requires Qt 4.6 and PyQt 4.6\n')
     sys.stderr.write('You have Qt %s and PyQt %s\n' %
                      (QT_VERSION_STR, PYQT_VERSION_STR))
     sys.exit()
